test/test_helper.exs

```diff
@@ -13,7 +13,7 @@ case :gen_tcp.connect('localhost', 4222, [:binary]) do
 end
 
 # this is used by some property tests, see test/gnat_property_test.exs
-Gnat.start_link(%{}, [name: :test_connection])
+Gnat.start_link(%{verbose: true}, [name: :test_connection])
```

Comment:
Why did we want this connection to run in verbose mode? I would think that because we are using this test for some property tests, we would want it to have minimal overhead?